README.md

```diff
@@ -1,11 +1,12 @@
-# Synergy Design System
+# Synergy
 
 <picture>
   <source media="(prefers-color-scheme: dark)" srcset="./packages/docs/public/synergy_logo_dark.png" width="400">
   <img alt="Logo" src="./packages/docs/public/synergy_logo_light.png" width="400">
 </picture>
 
-Welcome to the home of the Synergy Design System.
+Welcome to the home of the Synergy. Synergy is the foundation to efficiently build a consistent digital landscape for SICK.
```

Comment:
There is a typo here (at least, if we want to drop the "Design System" suffix)

Welcome to the home of ~~the~~ Synergy